app/controllers/settings/exports/following_accounts_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Settings
   module Exports
-    class FollowingAccountsController < ApplicationController
+    class FollowingAccountsController < BaseController
```

Comment:
ok, if this inherits from Settings::BaseController, doesn't that still require_functional!?